kiwi/filesystem/isofs.py

```diff
@@ -15,7 +15,7 @@
 # You should have received a copy of the GNU General Public License
 # along with kiwi.  If not, see <http://www.gnu.org/licenses/>
 #
-import os
+from distutils.spawn import find_executable
```

Comment:
ok, I compared the python packaging in RH and others and you are right there are big differences. I thought there is a global definition what is covered by python core but it seems every distro packages it differently. Looks like we need our own method to do the search as you suggested